lerna-util-sequence/src/main/resources/reference.conf

```diff
@@ -13,6 +13,9 @@ lerna.util.sequence {
   # 確保したシーケンスをインメモリに持つ、ワーカーを開放するまでの時間
   worker.idle-timeout = 10s
 
+  worker.stash-capacity = 1000
+  store.stash-capacity = 1000
```

Comment:
@tksugimoto 

[MUST] 設定項目の説明が必要かと思います。

**書く内容の例**
- どういうときに設定値を変更する必要があるのか
- 値の変更がコンポーネントの振る舞いにどう影響するのか
- 設定値が小さすぎるとどういうリスクがあるのか
- 設定値が大きすぎるとどういうリスクがあるのか

Reply:
対応しました。

174bb9f22b892e929b0f6cf2ea236ee95de9f302 `docs: lerna-util-sequence の stash-capacity conf に説明を追加`

----

参考: buffer があふれると、`StashOverflowException` が throw され、 Supervisor  によって restart されます（レスポンスなし)

```
ERROR lerna.util.sequence.SequenceStore$ | Supervisor RestartSupervisor saw failure: Couldn't add [lerna.util.sequence.SequenceStore$ReserveSequence] because stash with capacity [1] is full
akka.actor.typed.javadsl.StashOverflowException: Couldn't add [lerna.util.sequence.SequenceStore$ReserveSequence] because stash with capacity [1] is full
INFO  lerna.util.sequence.SequenceStore$ | created
INFO  lerna.util.sequence.SequenceStore$ | Cassandra session was ready (sequenceId: 8a156755-d19a-4c3b-950d-f4fb0cdbfd00, nodeId: 1)
```